src/client/process.ts

```diff
@@ -122,16 +122,34 @@ export class PowerShellProcess {
         return this.sessionDetails
     }
 
-    public async showTerminalIfVisible() {
+    public async showTerminalIfNotVisible() {
         if (this.consoleTerminal) {
             const winid: number = await vscode.workspace.nvim.eval(`bufwinid(${this.consoleTerminal.bufnr})`) as number;
 
-            // If winid is -1, it means the window is not visible/is hidden.
-            if (winid > -1) {
-                this.consoleTerminal.show(!this.config.integratedConsole.focusConsoleOnExecute);
+            // Show terminal if it's hidden when running F5/F8 or if focusConsoleOnExecute,
```

Comment:
remove explicit key bindings in comments

Reply:
changed!